docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -178,6 +178,33 @@ How to **fix** it:
 2. **Scale up distributors**<br />
    Scaling up distributors will lower the number of in-flight push requests per distributor.
 
+### MimirIngesterReachingInflightPushRequestLimit
+
+This alert fires when the `cortex_ingester_inflight_push_requests_summary` per ingester instance limit is enabled and the actual number of in-flight push requests is approaching the set limit. Once the limit is reached, push requests to the ingester will fail (5xx) or 429 for new requests, while existing in-flight push requests will continue to succeed.
+
+In case of **emergency**:
+
+- If the actual number of in-flight push requests is very close to or already at the set limit, then you can increase the limit via CLI flag or config to gain some time
+- Increasing the limit will increase the number of in-flight push requests which will increase ingesters' memory utilization. Please monitor the ingesters' memory utilization via the `Mimir / Writes Resources` dashboard
+
+How the limit is **configured**:
+
+- The limit can be configured either by the CLI flag (`-ingester.instance-limits.max-inflight-push-requests`) or in the config:
```

Comment:
I would suggest using runtime config file instead, which is reloaded by ingesters (unlike "normal" config file).